ql/src/java/org/apache/hadoop/hive/ql/ddl/database/drop/DropDatabaseAnalyzer.java

```diff
@@ -60,7 +62,13 @@ public void analyzeInternal(ASTNode root) throws SemanticException {
     // if cascade=true, then we need to authorize the drop table action as well, and add the tables to the outputs
     boolean isDbLevelLock = true;
     if (cascade) {
+      SessionState sessionState = SessionState.get();
       try {
+        HiveConf hiveConf = new HiveConf(conf);
+        hiveConf.set("hive.metastore.client.filter.enabled", "false");
+        if (sessionState != null) {
```

Comment:
Not sure if the current `db` would take this property, as this `db` is created by the original `conf` which doesn't have this property.

Reply:
Ack.